packages/shared/src/utils/contract/contract.ts

```diff
@@ -11,6 +11,17 @@ type TGetAccuBarriersDTraderTimeout = (params: {
     underlying: string;
 }) => number;
 
+// Trade types that are considered as vanilla financials
+export const VANILLA_FX = [
+    'frxAUDUSD',
+    'frxEURUSD',
+    'frxGBPUSD',
+    'frxUSDCAD',
+    'frxUSDJPY',
+    'frxXAUUSD',
+    'frxXAGUSD',
+] as const;
```

Comment:
Question: Why do we hard-code the trade types in FE code?

Reply:
There's no differentiation between vanilla options on synthetics or financials, the only way to know it is from symbol name. There was a decision long time ago, to handle these symbols on FE side.